src/installer/installer_test.go

```diff
@@ -24,15 +25,17 @@ func TestValidator(t *testing.T) {
 
 var _ = Describe("installer HostRoleMaster role", func() {
 	var (
-		l            = logrus.New()
-		ctrl         *gomock.Controller
-		mockops      *ops.MockOps
-		mockbmclient *inventory_client.MockInventoryClient
-		mockk8sclien *k8s_client.MockK8SClient
-		i            *installer
-		bootstrapIgn = "bootstrap.ign"
-		masterIgn    = "master.ign"
-		workerIgn    = "worker.ign"
+		l                = logrus.New()
+		ctrl             *gomock.Controller
+		mockops          *ops.MockOps
+		mockbmclient     *inventory_client.MockInventoryClient
+		mockk8sclien     *k8s_client.MockK8SClient
+		i                *installer
+		bootstrapIgn     = "bootstrap.ign"
+		masterIgn        = "master.ign"
+		workerIgn        = "worker.ign"
+		openShiftVersion = "4.4"
+		rhcos, _         = utils.GetRhcosImageByOpenshiftVersion(openShiftVersion)
```

Comment:
```suggestion
		image, _         = utils.GetRhcosImageByOpenshiftVersion(openShiftVersion)
```

Reply:
:) it was image and i changed it to rhcos